package-lock.json

```diff
@@ -1,8 +1,3482 @@
 {
   "name": "terraform",
   "version": "2.12.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Happy to upgrade - Just for posterity - which version of NPM do you have installed locally?

I wish there was a way of pinning it, like Node version via https://github.com/hashicorp/vscode-terraform/blob/main/.nvmrc 🤔

Reply:
I have version 7.6.3, a bit old 🙈

Reply:
which is still newer than mine (6.14.8) 😂  🙈 

... which explains the `lockfileVersion` bump I guess ...